vignettes/adrs.Rmd

```diff
@@ -0,0 +1,597 @@
+---
+title: "Creating ADRS"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating ADRS}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This article describes creating an `ADRS` ADaM with common oncology endpoint parameters
+based on RECIST v1.1. Further parameters could be added or users could use the function
+arguments to adapt the endpoints for different indications and response criteria.
```

Comment:
Maybe we should remove this sentence as it may raise too high expectations. I think we should explicitly state the response values (`CR`, `PR`, `SD`, ...) we expect and that these can not be modified at the moment. For example, we can not handle `iCR`.

Reply:
fixed